src/ConstructiveSolidGeometry/VolumePrimitives/VolumePrimitives.jl

```diff
@@ -15,6 +15,16 @@ function sample(vp::AbstractVolumePrimitive)
     unique!(mapreduce(sample, vcat, ls))
 end
 
+function _boundary_from_symbol(boundary::Symbol)
+    return if (boundary == :closed)
+        ClosedPrimitive
+    elseif (boundary == :open)
+        OpenPrimitive
+    else
+        error("Only `:closed` and `:open` are viable boundary options for CSG constructors")
+    end
+end
```

Comment:
Please remove this. We can just use `ClosedPrimitive` or `OpenPrimitive` directly.

We could change `CO` to `B` for boundary in general. But lets stick to `CO` in this PR (dedicated PR in case we want to).